products/names.json

```diff
@@ -2,30 +2,30 @@
   "akamai-base": "Akamai SDK",
   "akamai-edgekv": "Akamai SDK for EdgeKV",
   "android": "Android SDK",
-  "apex-server-sdk": "Apex Server SDK",
+  "apex-server-sdk": "Apex SDK",
   "cloudflare": "Cloudflare SDK",
-  "cpp-client-sdk": "C++ Client SDK",
-  "cpp-server-sdk": "C++ Server SDK",
-  "dotnet-client-sdk": ".NET Client SDK",
-  "dotnet-server-sdk": ".NET Server SDK",
+  "cpp-client-sdk": "C++ (client-side) SDK",
```

Comment:
The [linter error](https://github.com/launchdarkly/sdk-meta/actions/runs/8945849727/job/24575525153?pr=66) makes is sound like `()` are not allowed in this string?

Reply:
I can fix this in another PR